tests/test_kx.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+import nacl.bindings as b
+import nacl.exceptions as exc
+
+
+@pytest.mark.parametrize(("seed1", "seed2"), [
```

Comment:
I think it would be better to just generate some random seeds couples via hypothesis' `@given` like we already do in `tests/test_bindings.py`:
```
@given(binary(min_size=32,
              max_size=32),
       binary(min_size=32,
              max_size=32),
       )
@settings(max_examples=20)
def test_crypto_kx_keypair(seed1, seed2):
    ". . ."
```
for the standard case, and a couple of explicit failures cases